sdks/python/apache_beam/runners/worker/statesampler.py

```diff
@@ -49,18 +67,43 @@ def __init__(self, prefix, counter_factory,
     self.sampling_period_ms = sampling_period_ms
     super(StateSampler, self).__init__(sampling_period_ms)
 
+    # TODO(pabloem) - Remove this once all clients register the
+    # sampler independently.
+    self.register()
```

Comment:
As I mentioned before, I'd really like to avoid being in this state, even temporarily.